test/unit/extract/eia_bulk_elec_test.py

```diff
@@ -65,7 +65,7 @@ def test__parse_data_column(elec_txt_dataframe):
                 4371683.38189,
             ],
         },
-    )
+    ).convert_dtypes()
```

Comment:
Tried changing this `convert_dtypes()` to `apply_pudl_dtypes()` and it failed, so leaving it like this.